src/api/posts/server.js

```diff
@@ -0,0 +1,10 @@
+const { Satellite } = require('@senecacdot/satellite');
+const posts = require('./src/routes/posts');
+
+const service = new Satellite();
+
+service.router.use('/posts', posts);
```

Comment:
You can do this at the root `/` vs. `/posts` since Traefik will add a `/v1/posts` prefix for you.

Reply:
Didn't know about that, nice to know, thanks!